### PR TITLE
Migrate nextui from v1 to v2

### LINKED ISSUE #113: Switch from Material to NextUI

### PR DESCRIPTION
**Pull request by norbye**
v2 has _one_ component I wanted to use - so here we go..

Changes while I was at it;
* The GUI now displays the signature, with the option to remove it
* Removed the last remnants of Material UI
* Switched most styling to tailwind as it is required by nextui v2 so we might as well use it

Before can be seen at kvittering.abakus.no

## After
image

image

image

image

image

Resolves ABA-765
Closes #113

### CODE REVIEW

File: webapp/components/Sign.tsx [hunk 1]
```diff
@@ -42,50 +26,47 @@ const Sign = ({ modalIsOpen, setIsOpen, setSignature }: Props): JSX.Element => {
...
+          <div className={'flex p-3 gap-2 flex-col items-center'}>
+            <h3>Signer i feltet under</h3>
+            <p
+              style={{
+                margin: 0,
+                fontStyle: 'italic',
+                fontSize: '10px',
+                marginBottom: '-5px',
+              }}
+            >
```
Comment (haukkagu) [inline webapp/components/Sign.tsx:47]: Any reason why this paragraph doesn't use tailwind styling?

File: webapp/components/SignatureUpload.tsx [hunk 2]
```diff
@@ -40,41 +42,60 @@ const SignatureUpload = ({
...
+
+        <Button
+          className={'py-6 px-8'}
+          id="signButton"
+          isIconOnly
+          onPress={onOpen}
+          variant="ghost"
+          startContent={
+            <div className={'flex flex-col gap-1 items-center'}>
+              <p style={{ fontSize: '10px', margin: 0 }}>Eller tegn</p>
```
Comment (haukkagu) [inline webapp/components/SignatureUpload.tsx:69]: Same for this

**Review by haukkagu — APPROVED** [review]
Nice!

File: webapp/components/Sign.tsx (hunk 1 above)
Comment (norbye) [reply webapp/components/Sign.tsx:47]: Nope, I just missed it - will fix

**Review by ivarnakken — APPROVED** [review]
Awesome!

**Pull request merged**